app/cmd/mockify.go

```diff
@@ -23,19 +24,40 @@ type Route struct {
 }
 
 type Response struct {
-	URI         string            `yaml:"uri" json:"uri"`
-	Method      string            `yaml:"method" json:"method"`
-	RequestBody string            `yaml:"requestBody" json:"requestBody"`
-	StatusCode  int               `yaml:"statusCode" json:"statusCode"`
-	Headers     map[string]string `yaml:"headers" json:"headers"`
-	Body        interface{}       `yaml:"body" json:"body"`
+	URI           string            `yaml:"uri" json:"uri"`
+	Method        string            `yaml:"method" json:"method"`
+	RequestHeader string            `yaml:"requestHeader" json:"requestHeader"`
+	RequestBody   string            `yaml:"requestBody" json:"requestBody"`
+	StatusCode    int               `yaml:"statusCode" json:"statusCode"`
+	Headers       map[string]string `yaml:"headers" json:"headers"`
+	Body          interface{}       `yaml:"body" json:"body"`
 }
 
-var ResponseMapping = make(map[string]Response)
-var Router = mux.NewRouter()
+var (
+	HighestPriorityResponseMappings = make(map[string]Response)
+	SecondPriorityResponseMappings  = make(map[string]Response)
+	LowestPriorityResponseMappings  = make(map[string]Response)
```

Comment:
I like this but its not real clear the differences in the mappings. I _think_ it means that `HighestPriorityResponseMappings` is for request mapping, `SecondPriorityResponseMappings` is for header mapping. Is that correct?

Reply:
@brianmoran  Correct. I used those names so it would be easier to add more functionality in the future with different priorities. However, now that I come to think about it, it would be better and clearer to name the maps what they will contain. Will change :)

Reply:
Done!